### PR TITLE
feat: replacing grafana rom stable to its own repo + additional chart maintainer

### PR DESCRIPTION
**Pull request by gkarthiks**
Signed-off-by: gkarthiks <github.gkarthiks@gmail.com>

This PR is to update the Grafana's dependent repository from **`stable`** to **`grafana`**.

## ⚠️  Major Changes
👉  Version of Grafana chart has been upgraded from `5.3.*` to `5.6.*`
👉  `values.yaml` file's default value for Grafana has been same even after the bump of grafana chart.

### Addresses: https://github.com/prometheus-community/helm-charts/issues/30

/cc @bismarck @gianrubio @vsliouniaev 
🗒️  I also added myself as an additional maintainer to the chart.

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml
Comment (scottrigby) [inline charts/kube-prometheus-stack/Chart.yaml:10]: IMO it would be great to have extra help, but up to the existing maintainers of this chart.

File: charts/kube-prometheus-stack/README.md
Comment (scottrigby) [inline charts/kube-prometheus-stack/README.md:220]: Per #25 this should have been one sentence per line, I believe even in a list. Would you mind fixing this while you're here?

**Review by scottrigby — COMMENTED** [review]
Otherwise looks great!

File: charts/kube-prometheus-stack/README.md
Comment (gkarthiks) [reply charts/kube-prometheus-stack/README.md:220]: on it.

Comment (gkarthiks) [reply charts/kube-prometheus-stack/README.md]: @scottrigby added this and pushed, how does it looks like?

```
- Grafana: The Grafana chart is more feature-rich that contains a sidecar that is able to load data sources and dashboards from configmaps deployed into the same cluster and more information on that can be found by visiting the [documentation for the chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana)
```

Comment (scottrigby) [reply charts/kube-prometheus-stack/README.md]: I think I may be wrong about needing one sentence per line in a list. I'm not on my laptop now but from a quick GitHub flavored markdown test in a comment shows the line break is rendered if within a list... so maybe just keep as you had it.

Test:
- one sentence. Next sentence same line
- one sentence.
Next sentence on a new line

Comment (gkarthiks) [reply charts/kube-prometheus-stack/README.md]: @scottrigby 

there’s an issue even with the new word counts all over.
The default statement length seems to be “80” words, not sure we want to go with that count across the community.

This is the error with my new statement
```
README.md:220:81 MD013/line-length Line length [Expected: 80; Actual: 322]
```


for the original, it also errors about the new line as well.
```
README.md:219 sentences-per-line Each sentence should be on its own line [Context: "cluster. For m"]
README.md:220:81 MD013/line-length Line length [Expected: 80; Actual: 310]
```

So I am reverting to the old statement as of now.

I guess we have to agree unanimously what’s the statement’s word count should be on a single sentence. Then tackle on this issue. Thoughts ?

Comment (scottrigby) [reply charts/kube-prometheus-stack/README.md]: I think the original statement you had is great 👍 And yes, probably best to follow up about these questions on #25. Thank you!

**Review by scottrigby — APPROVED** [review]
This looks great 💯  Now just waiting for @bismarck @gianrubio or @vsliouniaev to approve adding you as an additional maintainer before merging.

**Comment by scottrigby** [comment]
Also related to #21

**Comment by torstenwalter** [comment]
@gkarthiks Can you resolve merge conflicts?

**Comment by gkarthiks** [comment]
@torstenwalter resolved the merge conflicts.

**Review by torstenwalter — COMMENTED** [review]
For me this looks good. Just needs an approval from one of the maintainers.

**Comment by gianrubio** [comment]
looks good to me!

**Review by gianrubio — APPROVED** [review]

**Pull request merged**
